lrpar/src/lib/ctbuilder.rs

```diff
@@ -691,16 +670,17 @@ C : 'a';"
                 .as_bytes()
         );
 
-        match CTParserBuilder::new()
+        let mut ct = CTParserBuilder::new()
             .error_on_conflicts(false)
-            .action_kind(ActionKind::GenericParseTree)
-            .process_file_in_src(file_path.to_str().unwrap())
-        {
-            Ok((_, conflicts)) => {
-                assert_eq!(conflicts.as_ref().unwrap().conflicts().sr_len(), 1);
-                assert_eq!(conflicts.as_ref().unwrap().conflicts().rr_len(), 1);
+            .action_kind(ActionKind::GenericParseTree);
+        let _ = ct.process_file_in_src(file_path.to_str().unwrap());
```

Comment:
`let _ = ` can be removed.

Reply:
Not without creating a warning tho.

Reply:
You can fix it in nicer ways than this ;)